src/lib/docs.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import { logInfoMessage, logError } from './logger.js'
+import Generator from '@asyncapi/generator'
+
+export default async (spec, config, resDir) => {
+  const configData = config.docs
```

Comment:
@AceTheCreator I don't see this configuration documented anywhere. am I missing something?

Reply:
@AceTheCreator you can update the type here https://github.com/asyncapi/glee/blob/0d4ff218ad2fec369fb77a9fcc6496424b292ef5/src/lib/index.d.ts#L36-L40

Reply:
@AceTheCreator I think you missed this conversation.